spec/std/string_spec.cr

```diff
@@ -1306,4 +1306,16 @@ describe "String" do
   it "gets length of \0 string" do
     "\0\0".length.should eq(2)
   end
+
+  context "%" do
+    it "substitutes one placeholder" do
+      res = "change %{this}" % { "this": "nothing" }
+      res.should eq "change nothing"
+    end
+
+    it "substitutes multiple placeholder" do
+      res = "change %{this} and %{more}" % { "this": "nothing", "more": "something" }
+      res.should eq "change nothing and something"
+    end
+  end
```

Comment:
What do you mean by "no hash is passed"? Something like `"change %{this}" %`? It seems a bit of an odd situation. How should the language behave?